src/pages/kubecon-europe-2020.js

```diff
@@ -0,0 +1,141 @@
+              >
+                Register here
+                <FeatherIcon
+                  className={styles.externalLinkIcon}
+                  name="external-link"
+                />
+              </Button>
+            </div>
+            <img
+              className={styles.img}
+              src={kubeconHeader}
+              alt="KubeCon and CloudNativeCon Europe 2020"
+            />
+          </section>
+          <section className={styles.section}>
+            <h2>Attend one of our lightning talks</h2>
+          </section>
+          <section className={cx(styles.section, styles.assetTable)}>
+            <div className={styles.point}>
+              <h4>We handle Prometheus, you keep Grafana</h4>
+              <p>Mon Aug 17 @ 14:35 CEST</p>
+              <p>Samuel Vandamme</p>
+            </div>
+            <div className={styles.point}>
+              <h4>How to use and customize Helm charts</h4>
+              <p>Mon Aug 17 @ 16:25 CEST</p>
+              <p>Douglas Camata</p>
+            </div>
+            <div className={styles.point}>
+              <h4>Kubernetes observability with context</h4>
+              <p>Tue Aug 18 @ 15:05 CEST</p>
+              <p>Stijn Polfliet</p>
+            </div>
+            <div className={styles.point}>
+              <h4>What is OpenTelemetry and how to get started?</h4>
+              <p>Tue Aug 18 @ 17:15 CEST</p>
+              <p>Lavanya Chockaligam</p>
+            </div>
+            <div className={styles.point}>
+              <h4>How to use and customize Helm charts</h4>
+              <p>Wed Aug 19 @ 15:05 CEST</p>
+              <p>Douglas Camata</p>
+            </div>
+            <div className={styles.point}>
+              <h4>OpenTelemetry Architecture</h4>
+              <p>Wed Aug 19 @ 16:25 CEST</p>
+              <p>John Watson</p>
+            </div>
+            <div className={styles.point}>
+              <h4>Kubernetes in the wild: best practices</h4>
+              <p>Thu Aug 20 @ 15:05 CEST</p>
+              <p>Martin Fuentes</p>
+            </div>
+            <div className={styles.point}>
+              <h4>Kubernetes observability with context</h4>
+              <p>Thu Aug 20 @ 16:50 CEST</p>
+              <p>Stijn Polfliet</p>
+            </div>
+
+            <div className={styles.highlightDiv}>
+              <FeatherIcon
+                className={styles.pointIcon}
+                name="search"
+                size="2rem"
+              />
+              <h4 className={styles.highlightText}>
+                Go to the virtual expo tab, and find New Relic in Silver Hall B
+                to attend a lightning talk.
+              </h4>
+            </div>
+          </section>
+          <section className={styles.section}>
+            <h2>Want some action now? Check out the following videos!</h2>
```

Comment:
Given this title, I'd expect these to be previews of the lightning talks that people will see at the conference.

Is that true? If not, I'd suggest making it clear that these videos are unrelated to the conference. For example:

`Start your learning journey before Kubecon starts`

Reply:
Yes these videos are related